src/io/fileops.c

```diff
@@ -246,6 +246,35 @@ MVMint64 MVM_file_isexecutable(MVMThreadContext *tc, MVMString *filename, MVMint
     }
 }
 #else
+
+static int are_we_group_member(MVMThreadContext *tc, gid_t group) {
+    int len;
+    gid_t *gids = NULL;
+    int res;
+    int i;
+    // Check the user group.
```

Comment:
I thought these style comments actually caused problems in CI somewhere, but regardless, I believe there is at least a convention to only use ‘/* */‘ comments.